lib/syskit/cli/log_runtime_archive.rb

```diff
@@ -12,17 +13,27 @@ module CLI
         # It depends on the syskit instance using log rotation
         class LogRuntimeArchive
             DEFAULT_MAX_ARCHIVE_SIZE = 10_000_000_000 # 10G
+            FREE_SPACE_LOW_LIMIT = 5_000_000_000 # 5 G
+            FREE_SPACE_FREED_LIMIT = 25_000_000_000 # 25 G
 
             def initialize(
                 root_dir, target_dir,
                 logger: LogRuntimeArchive.null_logger,
                 max_archive_size: DEFAULT_MAX_ARCHIVE_SIZE
```

Comment:
What you did here was actually not what I meant.

Currently, you let the user set instance-wide defaults for free space thresholds **and** override these defaults in the method. I think that's a tad bit too much.

What I meant was that you could remove the instance-wide defaults, keeping only the arguments to ensure_free_space.

(Which makes me realize that the (low_limit < freed_limit check should also be done in ensure_free_space)

Reply:
oh, got it now. fixed.